script/_dhcpv6_pd_ref

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+customise_dhcpcd_conf()
+{
+    sudo tee /etc/dhcpcd.conf >/dev/null <<EOF
+# A sample configuration for dhcpcd.
+# See dhcpcd.conf(5) for details.
+
+# Allow users of this group to interact with dhcpcd via the control socket.
+#controlgroup wheel
+
+# Inform the DHCP server of our hostname for DDNS.
+#hostname
+
+# Use the hardware address of the interface for the Client ID.
+#clientid
+# or
+# Use the same DUID + IAID as set in DHCPv6 for DHCPv4 ClientID as per RFC4361.
+# Some non-RFC compliant DHCP servers do not reply with this set.
+# In this case, comment out duid and enable clientid above.
+#duid
+
+# Rapid commit support.
+# Safe to enable by default because it requires the equivalent option set
+# on the server to actually work.
+option rapid_commit
+
+# A ServerID is required by RFC2131.
+#require dhcp_server_identifier
+
+noipv6rs 
+# disable routing solicitation
+# denyinterfaces eth1  # Don't touch other_interfaces at all
+interface eth0
+  ipv6rs
+  iaid 1
+  ia_pd 2/::/64 -
```

Comment:
When trying out this PR, we noticed that there were two lines `noipv6` and `noipv6rs` at the end of this file. Not sure how they were added.

Reply:
Probably here: https://github.com/openthread/ot-br-posix/blob/ff7227ea9a2dc0e50f92ab93bd5c9a29e82fe793/script/_border_routing#L84

Reply:
the outside noipv6 and noipv6rs was from the default config, inside the request RA config (which under interface eth0) we were requesting /64 prefix, so it should be fine

Reply:
I don't know. Given the default config has both `noipv6` and `noipv6rs`, doesn't it need both `ipv6` and `ipv6rs` for the interface?

Reply:
the outside noipv6 and noipv6rs won't affect the block of "request RA" config